src/process_utility.c

```diff
@@ -1618,7 +1617,7 @@ timescaledb_ddl_command_start(
 	 * If we are restoring, we don't want to recurse to chunks or block
 	 * operations on chunks. If we do, the restore will fail.
 	 */
-	if (!extension_is_loaded() || guc_restoring)
+	if (!extension_is_loaded())
```

Comment:
Maybe clarify comment above, since it is no longer clear that this is related to restoring?